src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx

```diff
@@ -803,7 +811,7 @@ const mockRequestResponses = {
           message_header: "Success",
           message_description_md: "",
           position: {
-            internalID: "some-bidder-position-id",
+            internalID: "bidder-position-id-from-mutation",
```

Comment:
what's the level of effort requested to refactor the `pollingForBid` fixture to use relay like this fixture? Could we resolve these TODOs in this PR? Seems like that would increase the robustness of these tests for any future refactorings

Reply:
The expectations for polling was simply missing and this basically backfills test coverage. The first exception specifically tests that the `internalID` from the mutation results is properly passed on to the first polling request:

https://github.com/artsy/emission/blob/ec0e94d6403122496c4ac3eb226378d16588377b/src/lib/Components/Bidding/Screens/ConfirmBid.tsx#L297

And the second one specifically tests the handling of the one from the polling results:

https://github.com/artsy/emission/blob/ec0e94d6403122496c4ac3eb226378d16588377b/src/lib/Components/Bidding/Screens/ConfirmBid.tsx#L317

While in production both calls would return the exact same id, I've used two different ids in test to make sure each one of the calls works as expected.

Reply:
This lead to a IRL conversation about the feasibility about generating types for these fixtures. See this comment: https://github.com/artsy/emission/pull/1744/files#r306406212

TL;DR is that we aren't going to worry about this type enhancement in this PR.